instrumentation/grpc-1.6/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_6/GrpcAttributesExtractor.java

```diff
@@ -14,7 +14,9 @@
 final class GrpcAttributesExtractor implements AttributesExtractor<GrpcRequest, Status> {
   @Override
   public void onStart(AttributesBuilder attributes, GrpcRequest grpcRequest) {
-    // No request attributes
+    if (grpcRequest.getAuthority() != null) {
+      attributes.put(GrpcHelper.RPC_GRPC_AUTHORITY, grpcRequest.getAuthority());
```

Comment:
This new attribute does not seem to be present in the [RPC semantic conventions spec](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/rpc.md#grpc). Would you mind opening a PR/issue against the spec repo that introduces this attribute?
And, for now, how about moving it to a separate `AttributesExtractor` that's disabled by default? We're using the `experimental-span-attributes` for these, e.g. `otel.instrumentation.aws-sdk.experimental-span-attributes`

Reply:
Note that I agree this attribute seems to already be captured as part of `net.peer` so unless there is a difference, we probably wouldn't want to go forward with even an experimental attribute.

Reply:
Thank you for your reply. I found in grpc client spans there were no `net.peer` attributes. So I add the `authority` attribute by this PR and then we can know the invoked remote service's host and port. Is this a bug in grpc client instrumentation?

Reply:
@ralphgj there should be `net.peer` attributes captured for grpc client spans since version 1.4.0 (#1338), if you're on latest and not seeing them it could be a bug

Reply:
thx @trask. I will close this PR and comment in #1338